geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -228,6 +230,30 @@ private void validateScoreIsDouble(byte[] score) {
     return members.get(member);
   }
 
+  byte[] zincrby(Region<RedisKey, RedisData> region, RedisKey key, byte[] increment,
+      byte[] member) {
+    byte[] score = members.get(member);
+    byte[] incr = Coder.doubleToBytes(
+        processIncrement(Coder.bytesToString(increment).toLowerCase()));
+
+    if (Arrays.equals(incr, ERROR_NOT_A_VALID_FLOAT.getBytes())) {
+      return incr;
+    }
```

Comment:
Since `processIncrement` is just throwing an exception now, this check and be removed.